src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -28,154 +26,203 @@
-        arm.setAngle(Util.map(armAnalogInput.getValue(), Constants.ARM_MIN, Constants.ARM_MAX, 180, 0));
-        wrist.setAngle(Util.map(wristAnalogInput.getValue(), Constants.WRIST_MIN, Constants.WRIST_MAX, 90, -90));
-    }
 
-    public void setArmPoistion(double armAngle) {
-        armPIDController.setSetpoint(armAngle);
-        SmartDashboard.putNumber("Arm Setpoint", armAngle);
-    }
+    armFalcon = new WPI_TalonFX(Constants.ARM_FALCON_ID);
+    armFalcon.setNeutralMode(NeutralMode.Coast);
 
-    public void adjustWristLevel(double offsetAdjustment) {
-        wristOffset += offsetAdjustment;
-    }
+    wristFalcon = new WPI_TalonFX(Constants.WRIST_FALCON_ID);
+    wristFalcon.setNeutralMode(NeutralMode.Coast);
 
-    public double getArmPosition() {
-        return armAnalogInput.getValue();
-    }
+    armPIDController = new PIDController(.001, 0, 0);
+    wristPIDController = new PIDController(.001, 0, 0);
 
-    public double getArmSetpoint() {
-        return armPIDController.getSetpoint();
-    }
+    wristPIDController.setSetpoint(0);
+    armPIDController.setSetpoint(Constants.CUBE_LOW_ANGLE);
 
-    public double getWristPosition() {
-        return wristAnalogInput.getValue();
+    this.wristOffset = Constants.NEUTRAL_WRIST_OFFSET;
+
+    this.brakeSolenoid = ph.makeSolenoid(Constants.ARM_BRAKE_ID);
+    setBrakeOn();
+
+    this.mechanism = new Mechanism2d(5, 5);
+    this.root = mechanism.getRoot("support", 4, 0);
+    this.support = root.append(new MechanismLigament2d("support", 2, 90));
+    this.arm = support.append(new MechanismLigament2d("arm", 2, 135));
+    this.wrist = arm.append(new MechanismLigament2d("wrist", 1, -45));
+
+    this.armSim = new SingleJointedArmSim(
+        DCMotor.getFalcon500(1),
+        300,
+        10,
+        1,
+        0,
+        3,
+        1,
+        true);
+
+    this.wristSim = new SingleJointedArmSim(
+        DCMotor.getFalcon500(1),
+        100,
+        1,
+        0.1,
+        -3,
+        7,
+        1,
+        false);
+
+    if (Robot.isSimulation()) {
+      armAnalogInputSim.setVoltage(0.3);
+      wristAnalogInputSim.setVoltage(0.7);
     }
 
-    @Override
-    public void simulationPeriodic() {
-        armSim.setInputVoltage(armFalcon.getMotorOutputVoltage());
-        wristSim.setInputVoltage(wristFalcon.getMotorOutputVoltage());
-        armAnalogInputSim.setVoltage(Util.map(armSim.getAngleRads(), 0, Math.PI, 0.3, 2.3));
-        wristAnalogInputSim.setVoltage(Util.map(wristSim.getAngleRads(), -Math.PI/2, Math.PI/2, 2.1, 4.8));
+    this.setName("ArmSubsystem");
+  }
+  
+  public void brakeOpen () {
+  brakeSolenoid.set(false);
+  }
+
+  public void brakeClosed () {
+  brakeSolenoid.set(true);
+  }
+
+  @Override
+  public void periodic() {
+    double armPower = armPIDController.calculate(armAnalogInput.getValue());
+    double wristPower = wristPIDController.calculate(wristAnalogInput.getValue()+armAnalogInput.getValue()-wristOffset);
```

Comment:
Check the input for this PID controller. I don't think you want the arm position in the calculation for the wrist.